### ISSUE TITLE
tf.train.shuffle_batch

### ISSUE THREAD

**Issue by zhangsj0608** [zhangsj0608/distributed_tf#1]
tf.train.shuffle_batch接受tensor的列表，怎样入队？shuffle？生成batch？生成的图为何可以不断从队列中取出tensor的值（queue_runner），当队列取完的时候，入队操作是否重新打开文件，将文件内容入队(出现多次重复数据的原因)?
